internal/pkg/skuba/addons/cilium.go

```diff
@@ -60,20 +60,15 @@ func (ciliumCallbacks) beforeApply(addonConfiguration AddonConfiguration, skubaC
 	if err != nil {
 		return errors.Wrap(err, "unable to determine if cilium secret exists")
 	}
-	ciliumConfigMapExists, err := cni.CiliumConfigMapExists(client)
```

Comment:
This function can be removed as well now that it has no callers.

Reply:
good catch, removed!